examples/attention/SentenceGenStimulateAgent.cc

```diff
@@ -0,0 +1,110 @@
+/*
+ * SentenceGenAgent.cc
+ *
+ *  Created on: 10 Nov 2015
+ *      Author: misgana
+ */
+#include <sstream>
+#include <fstream>
+#include <chrono>
+#include <thread>
+#include <boost/filesystem.hpp>
+
+#include <opencog/atomutils/AtomUtils.h>
+#include <opencog/attention/atom_types.h>
+#include <opencog/cogserver/server/Factory.h>
+#include <opencog/nlp/types/atom_types.h>
+
+#include "ExperimentSetupModule.h"
+#include "SentenceGenStimulateAgent.h"
+
+using namespace opencog;
+using namespace opencog::ECANExperiment;
+
+using namespace std;
+using namespace chrono;
+
+SentenceGenStimulateAgent::~SentenceGenStimulateAgent()
+{
+    //delete _scm_eval;
+}
+SentenceGenStimulateAgent::SentenceGenStimulateAgent(CogServer& cs) :
+        Agent(cs), _as(cs.getAtomSpace())
+{
+    _scm_eval = new SchemeEval(&_as);
+    current_group = 0;
+    startcount = _cogserver.getCycleCount();
```

Comment:
Do cogserver agents actually work? Last time I tried them, they didn't work; and fixing them looked to be a major undertaking...  I don't believe that infrastructure is currently usable.